### PR TITLE
Add MNSes for ISC 22

### PR DESCRIPTION
**Pull request by ocaisa**

### CODE REVIEW

**Comment by klust** [comment]
This text is becoming a little confusing as it currently mixes three foss versions. The graph is from 2020a (I suppose from the text, I didn't really check in the easyconfigs), one example is from 2020b and then everything else has been updated to 2021b.

Do we want this or should it be made a bit more uniform? I did figure out how to redo the figure (and tested that already). 

I can have a look at it but given that this pull request does not have a branch in this repository it might be easier to have it merged first and that I do a new pull request then with the changes? And then the slides should be checked to be consistent with the text. I do have some time this afternoon if needed.

**Comment by ocaisa** [comment]
@klust I would have updated the 2020b example for the MNS but the 2021b version of the SciPy-bundle no longer has a versionsuffix

File: docs/2022-isc22/module_naming_schemes.md [hunk 1]
```diff
@@ -1 +1,608 @@
...
+      module load SciPy-bundle/2021.10
+      ```
+    * Run the test command:
+      ```
+      $ python -c 'import pandas; print(pandas.__version__)'
+      1.3.4
+      ```
+---
+
+[*next: EasyBuild at Jülich Supercomputing Centre*](jsc.md) - [*(back to overview page)*](index.md)
```
Comment (ocaisa) [inline docs/2022-isc22/module_naming_schemes.md:608]: @SebastianAchilles This link is broken until we have a `jsc.md` file

**Comment by ocaisa** [comment]
@klust I've updated the 2020b example now but I would leave it to you to update the graph in a follow-up PR

**Comment by klust** [comment]
Better to merge I think so that I can start again from a file in the main repository that I cloned. But there is still a test failing.

**Comment by ocaisa** [comment]
Merging based on the review from @klust

**Pull request merged**